src/config_ctrl.ts

```diff
@@ -36,6 +37,10 @@ export class BigQueryConfigCtrl {
     if (this.current.jsonData.processingLocations === undefined) {
       this.current.jsonData.processingLocations = this.defaultProcessingLocation;
     }
+    if (this.current.jsonData.queryPriority === undefined) {
```

Comment:
Why not do `!this.current.jsonData.queryPriority` to also make sure empty value or something else

Reply:
It is coming from hard coded list,